### ISSUE TITLE
Final Project Step 01 Question

### ISSUE THREAD

**Issue by mtwelker** [DS4PS/cpp-524-sum-2021#9]
I have read the project instructions and watched the review session, but I'm a bit confused about exactly what we're supposed to turn in this week for step one.  Is it basically page one of what will be our final paper?  So is the expectation that this week we do the research outlined in the paper requirements ("...find at least two evaluation or performance studies that examine a similar kind of program model. Use the background research to see what evaluation design others have used to establish an estimate of program impact.") and use that to describe The Problem, Program, Program theory, etc., as outlined on the page one memo?
I guess I'm confused because the assignment also says that "This first step is designed to get feedback from the instructor in order to ensure you have picked an appropriate topic for the assignment."  And that seems like a lot of work to do if there's a chance that the topic we choose isn't appropriate for this assignment.
Thanks for any clarification you can offer!

**Comment by lecy** [comment]
Correct, just the first page with some parsimonious program information. 

You don't have to do extensive background research if you can provide some basic information about your program. However, if you are selecting a program you are unfamiliar with then you would need to find enough documentation to provide details. 

The instructor feedback is less about evaluating appropriateness of the program you have selected, and more ensuring your topic has proper scope and you have enough details to complete an evaluation design. If you can't answer general questions about creating a study sample you will struggle to build an evaluation for the program. 

The assignment focuses on how well you can articulate elements of the evaluation design, less on details of a specific program. In many cases you will need to provide some hypothetical information to provide sufficient detail for the study design. The goal is writing a coherent research proposal, not how precisely you have described an actual program. 

In most cases you want to describe the program in a generic sense anyhow. Think about your audience as a person with a general social science PhD working at a foundation to evaluate a broad portfolio of program. They are smart enough to understand the program generally but might not be an expert in the specific area. What sorts of details would be important to explain the program model and study sample? 

![image](https://user-images.githubusercontent.com/1209099/119240441-642f0b00-bb04-11eb-94c0-2e7c69431e9f.png)

**Comment by mtwelker** [comment]
That's very helpful.  Thank you!
